tests/test_polydata.py

```diff
@@ -172,6 +177,24 @@ def test_invalid_file():
         pyvista.PolyData(filename)
 
 
+def test_lines_on_init():
+    lines = [2, 0, 1, 3, 2, 3, 4]
+    points = np.random.random((5, 3))
+    pd = pyvista.PolyData(points, lines=lines)
+    assert not pd.faces.size
+    assert np.allclose(pd.lines, lines)
+    assert np.allclose(pd.points, points)
```

Comment:
```suggestion
    assert np.array_equal(pd.lines, lines)
    assert np.array_equal(pd.points, points)
```
`lines` is an int array so this has to be correct, but even for the float `points` array it would be really fishy if the values changed slightly (there shouldn't be any arithmetic involved).

Reply:
True, and actually, since points is not deep copied, the values *must* be identical hence with:
```py
>>> points[:] = 0
>>> pd.points
>>> pyvista_ndarray([[0., 0., 0.],
                     [0., 0., 0.],
                     [0., 0., 0.],
                     [0., 0., 0.],
                     [0., 0., 0.]])
```

Reply:
Then odds are `pd.points is points`.